tex/gloss-marathi.ldf

```diff
@@ -1,91 +1,97 @@
 % Translations provided by Abhijit Navale <abhi_navale@live.in>
+% Translations reviewed by Niranjan Tambe <niranjanvikastambe@gmail.com> on 13th December, 2019
 % TODO implement Hindu calendar (not used in day-to-day practice)
 
 \ProvidesFile{gloss-marathi.ldf}[polyglossia: module for marathi]
 \ifluatex
-  \xpg@warning{Marathi is not supported with LuaTeX.\MessageBreak
+\xpg@warning{Marathi is not supported with LuaTeX.\MessageBreak
 I will proceed with the compilation, but\MessageBreak
 the output is not guaranteed to be correct\MessageBreak
 and may look very wrong.}
 \fi
 \RequirePackage{devanagaridigits}
 
 \PolyglossiaSetup{marathi}{
-  bcp47=mr,
```

Comment:
please do not remove existing tags such as `bcp47` or `localnumeral`. I think you should not touch `\PolyglossiaSetup` at all-